grakn-bootup/src/main/java/ai/grakn/bootup/EngineProcess.java

```diff
@@ -47,6 +48,7 @@
     private static final String GRAKN_NAME = "Grakn";
     private static final long GRAKN_STARTUP_TIMEOUT_S = 300;
     public static final Path ENGINE_PID = Paths.get(File.separator,"tmp","grakn-engine.pid");
+    public static final Optional<String> javaOpts = Optional.ofNullable(System.getProperty("engine.javaopts"));
```

Comment:
There's a class `GraknSystemProperty` where we put all our system properties. This could go there too.

We could even add a new method there like `getOrDefault`, or one that returns an optional 👍

Reply:
fixed 👍